code_search/kubeflow/components/experiments.libsonnet

```diff
@@ -0,0 +1,11 @@
+// Data for various experiments.
+// Paths are deliberately hard coded so they get versioned and checked into source control.
```

Comment:
I really like this idea in combination with being able to construct a configuration in python, from a notebook.